# https://github.com/rpm-software-management/drpm/pull/24
From b1374da3d837906ac1ed36efd7b845e515640c40 Mon Sep 17 00:00:00 2001
From: Matt Jolly <kangie@gentoo.org>
Date: Wed, 23 Apr 2025 17:43:31 +1000
Subject: [PATCH] Update `cmake_minimum_required` to support modern CMake

CMake 4 has dropped support for compat below 3.5.0.

Since 3.5.0 was released in 2016, even LTS releases should include it
in 2025.

Use `<min>...<max>` to take into account the future removal of compat
support for <3.10.0.

Signed-off-by: Matt Jolly <kangie@gentoo.org>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d37272c..27a40f3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.5)
+cmake_minimum_required(VERSION 3.5.0...3.10.0)
 
 project(drpm C)
 
